### PR TITLE
feat: Adding view and delete card shipment to Node wrapper [ENGRTF-1944]

### PR DESCRIPTION
**Pull request by jasonto1**
Adding the capability to view all subnet shipments, view single subnet shipment, and deleting subnet shipment to wrapper to account for the new API endpoints below:

- https://docs.synapsefi.com/api-references/shipments/view-all-subnet-shipments
- https://docs.synapsefi.com/api-references/shipments/view-shipment
- https://docs.synapsefi.com/api-references/shipments/cancel-shipment

### CODE REVIEW

**Review by japamatSynapse — COMMENTED** [review]
Beautiful use of JSDoc comment structure, looks pretty straight forward. Will wait for @sheilf-synapse for final approval

**Review by sheilf-synapse — APPROVED** [review]

**Pull request merged**
